parser.go

```diff
@@ -862,6 +862,7 @@ func (parser *Parser) parseStructField(pkgName string, field *ast.Field) (map[st
 
 	structField, err := parser.parseField(field)
 	if err != nil {
+		Println(err)
```

Comment:
I guess you want to show error warning message here, right? 👍 Could you help fix to return an error there instead of ignoring it? 
As previously refactoring commits were not fixing well and I have also known that there were a few the same things in the code where not return an error. It's okay for this PR.

Reply:
yes. 
Modified, return error directly, no need to print there.